tests/fail/tree_borrows/alternate-read-write.rs

```diff
@@ -13,7 +13,8 @@ pub fn main() {
     // This time y gets Frozen...
     let _val = *x;
     // ... and the next Write attempt fails.
-    *y += 1; // Failure //~ ERROR: /write access through .* is forbidden/
+    //~v ERROR: /write access through .* is forbidden/
+    *y += 1; // Failure
```

Comment:
Why did this have to change? What happens when one does `// Failure //~ERROR ...`?

Reply:
We don't allow annotations in comments anymore and will error if they occur there.

It's annoying on the parser to allow this, as I can't just look for the first `//` and parse from there